content/layouts/page-breadcrumbs.html

```diff
@@ -17,12 +17,6 @@
     {% when '3-panel-filters' %}
```

Comment:
Since the `3-panel-filters` include is being deleted, can this be removed from the breadcrumbs too?

Reply:
Good call!